Engine/AppManagerPrivate.cpp

```diff
@@ -763,9 +769,13 @@ AppManagerPrivate::initGLAPISpecific()
     OSGLContext_win::initWGLData( wglInfo.get() );
 #elif defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD)
     // scoped_ptr
```

Comment:
(for later) we should change every occurence of boost::scoped_ptr to std::unique_ptr, since we're C++11.

Reply:
Yeah, I think this is going to be relatively easy as `std::unique_ptr` contains every method from `boost::scoped_ptr`.